mlflow/entities/trace.py

```diff
@@ -27,9 +28,13 @@ class Trace(_MlflowObject):
         data: A container object that holds the spans data of a trace.
     """
 
-    info: TraceInfo
+    info: TraceInfoV3
     data: TraceData
 
+    def __post_init__(self):
+        if isinstance(self.info, TraceInfo):
+            self.info = self.info.to_v3(request=self.data.request, response=self.data.response)
+
     def __repr__(self) -> str:
         return f"Trace(request_id={self.info.request_id})"
 
```

Comment:
Can we add a `"version": "3.0"` field to the `to_dict` result? This will help us to determine how to parse the JSON representation in downstream consumers.

Reply:
We have this tag for the schema version:

https://github.com/mlflow/mlflow/blob/e85be48e985d364b3b7c2e6d9a18cdc423cdec48/mlflow/tracing/constant.py#L47

Can we set this tag to 3?

Reply:
Thanks for the pointer, let's do that.